starter-code/index.html

```diff
@@ -22,17 +22,17 @@ <h1>Pizza Builder</h1>
         <li v-for="ingredient in ingredients">
           <!-- The button must have the `active` class if the ingredient is selected -->
           <!-- When the button is clicked, it should toggle the `selected` property of the corresponding ingredient object -->
-          <button
-            :class="[
+          <button @click="ingredient.selected? ingredient.selected=false : ingredient.selected=true" :class="[
               'btn',
-              'btn-' + ingredient.name.replace(' ', '-')
-            ]"
-          >
+              'btn-' + ingredient.name.replace(' ', '-'),
+              ingredient.selected? 'active': ''
```

Comment:
You could also use the slightly nicer object syntax: `{ active: ingredient.selected }`